apps/mobile-wallet/src/screens/new-wallet/ImportWalletSeedScreen.tsx

```diff
@@ -143,9 +142,10 @@ const ImportWalletSeedScreen = ({ navigation, ...props }: ImportWalletSeedScreen
           headerRight: () => <ContinueButton onPress={handleWalletImport} disabled={!isImportButtonEnabled} />
         }}
         keyboardShouldPersistTaps="always"
+        screenTitle="Secret phrase"
+        screenIntro={`Enter the secret phrase for the ${name} wallet.`}
```

Comment:
We lost the wrapping of the wallet name with `"` with this change.

Reply:
Compiler was complaining at one point. But you're right, adding them back now (no complains anymore).